CHANGELOG.md

```diff
@@ -12,6 +12,128 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 🛠 Maintenance
 ## 📚 Documentation -->
 
+# [0.0.5] - 2021-03-30
+
+> Important: X breaking changes below, indicated by **❗ BREAKING ❗**
+
+## 🚀 Features
+- **Adds introspection ability for subgraphs - [lrlna], [issue/349] [pull/377]**
+
+  A new command, `rover subgraph introspect` has been added. This command
+  runs a _federated introspection_ query against a server which has
+  implemented the requirements of the [federation
+  specification](https://www.apollographql.com/docs/federation/federation-spec/).
+  This command accepts endpoint [headers] for making the introspection
+  request (if required) and outputs SDL to stdout.
+
+  [lrlna]: https://github.com/lrlna
+  [pull/377]: https://github.com/apollographql/rover/pull/277
+  [issue/349]: https://github.com/apollographql/rover/issues/349
+
+- ** Fallback to monochromic output when `tput` is unavailable - [abernix], [issue/371] [pull/372]**
```

Comment:
```suggestion
- **Fallback to monochromic output in installer when `tput` is unavailable - [abernix], [issue/371] [pull/372]**
```

Reply:
not sure if we need the "in installer" but that space after the `**` will make the heading not bold